docs/APISocketOptions.md

```diff
@@ -1094,33 +1099,41 @@ The value set here will be effectively aligned to the multiple of payload size.
 
 | OptName           | Since | Binding | Type       |  Units  |   Default  | Range  | Dir | Entity |
 | ----------------- | ----- | ------- | ---------- | ------- | ---------- | ------ | --- | ------ |
-| `SRTO_RCVLATENCY` | 1.3.0 | pre     | `int32_t`  | msec    | *          | 0..    | RW  | GSD    |
+| `SRTO_RCVLATENCY` | 1.3.0 | pre     | `int32_t`  | ms      | *          | 0..    | RW  | GSD    |
 
-- Latency value in the receiving direction. This value is only significant when
-`SRTO_TSBPDMODE` is set to true.
+- The latency value in the receiving direction of the current SRT connection.
```

Comment:
Well, "of the current SRT connection" is not the happiest statement here, as it might be understood that you should set this option upon connection, which is actually the opposite of how you should use it.

Maybe `The latency value in the receiving direction to be used for this socket's connection` ?

Reply:
Changed to
> The latency value in the receiving direction of the socket.